barter-data/README.md

```diff
@@ -51,7 +51,7 @@ arbitrary number of exchange `MarketStream`s using input `Subscription`s. Simply
 |       **Bitmex**        |             `Bitmex`             |                  Perpetual                  |                   PublicTrades                   |
 |      **BybitSpot**      |      `BybitSpot::default()`      |                    Spot                     |                   PublicTrades                   |
 | **BybitPerpetualsUsd**  | `BybitPerpetualsUsd::default()`  |                  Perpetual                  |                   PublicTrades                   |
-|      **Coinbase**       |            `Coinbase`            |                    Spot                     |                   PublicTrades                   |
+|      **Coinbase**       |            `Coinbase`            |                    Spot                     |          PublicTrades <br> OrderBooksL1          |
```

Comment:
Should this include OrderBookL2? 

Disclaimer: this is at the top of the diff so sorry if this is correct, it's just the MR title mentiones L2